packages/opentelemetry-sdk-trace-base/src/export/SimpleSpanProcessor.ts

```diff
@@ -38,16 +37,16 @@ import { Resource } from '@opentelemetry/resources';
  */
 export class SimpleSpanProcessor implements SpanProcessor {
   private _shutdownOnce: BindOnceFuture<void>;
-  private _unresolvedExports: Set<Promise<void>>;
+  private _pendingExports: Set<Promise<unknown>>;
 
   constructor(private readonly _exporter: SpanExporter) {
     this._shutdownOnce = new BindOnceFuture(this._shutdown, this);
-    this._unresolvedExports = new Set<Promise<void>>();
+    this._pendingExports = new Set<Promise<void>>();
```

Comment:
nit: should this void be changed to unknown as well? I'm no TypeScript expert here.

```suggestion
    this._pendingExports = new Set<Promise<unknown>>();
```

Or, why was the s/void/unknown/ change made on line 41?

Reply:
Me neither - a previous version of the code was putting the result of internal._export which isn't `void` but eventually landed on the current one which didn't need the change. I reverted to reduce changes